api/validation.go

```diff
@@ -60,10 +60,20 @@ func (api *API) validateAndHydrateDimensions(v dataset.Version, dims []model.Dim
 	}
 
 	if v.IsBasedOn.Type == cantabularMultivariateTable {
```

Comment:
Do we want to deal multivariate differently than flexible, won't adding a 'or' condition for multivariate here would address it https://github.com/ONSdigital/dp-cantabular-filter-flex-api/blob/cbc959241993cb2477272c32b0dc911aa5d76cd4/api/validation.go#L25